### DISCUSSION TITLE
Create direct shortcut to popped-up mode window

### DISCUSSION THREAD

**Discussion by FaberfoX** [Authenticator-Extension/Authenticator#682] [Q&A] [answered]
**Describe the feature you want:**
I'd love to be able to launch the extension, in its popped up mode (like when double clicking the extension title) from a shortcut on the desktop or taskbar. Right now, if I right click and pin the window to taskbar on Windows 10, clicking on it after having closed it, opens a new tab with the scary "Reset Authenticator"

![image](https://user-images.githubusercontent.com/6801510/120747978-6025c480-c4d8-11eb-9ade-2bbff21c439b.png)

**Why do you want this feature in Authenticator?**
I'm trying to use Guacamole integrating it on mRemoteNG, but I need to keep a Chrome window open, or previously launch this in a pop up.

**Comment by Sneezry** [comment, marked as answer]
Not like Apps, Chrome doesn't support to create shortcut in desktop or taskbar for extensions.

However, you can use a tricky way to make it possible:

1. Open `chrome-extension://bhghoamapcdpbohphigoooaddinpkbai/view/popup.html?popup=true` in a new tab
2. Click **Create shortcut...** in **More tools** menu
   ![image](https://user-images.githubusercontent.com/1621293/120860672-a4889180-c5b8-11eb-8704-6d7f9e496ce4.png)
3. Check **Open as window** in the popup dialog
   ![image](https://user-images.githubusercontent.com/1621293/120860827-de599800-c5b8-11eb-8f98-423654b71ef9.png)
4. You will see Authenticator shortcut in your desktop. Open it, and pin it in taskbar.

The window size may be incorrect. And you need resize the window manually. The window size will be remembered after you close it.

**Comment by FaberfoX** [comment]
> Not like Apps, Chrome doesn't support to create shortcut in desktop or taskbar for extensions.
> 
> However, you can use a tricky way to make it possible:
> 
> 1. Open `chrome-extension://bhghoamapcdpbohphigoooaddinpkbai/view/popup.html?popup=true` in a new tab
> 2. Click **Create shortcut...** in **More tools** menu
> 3. Check **Open as window** in the popup dialog
> 4. You will see Authenticator shortcut in your desktop. Open it, and pin it in taskbar.
> 
> The window size may be incorrect. And you need resize the window manually. The window size will be remembered after you close it.

Beautiful, this is perfect, thanks a lot!
Closing this as it's somewhat solved taking Chrome's limitations into account.
